python/ray/serve/tests/test_autoscaling_policy.py

```diff
@@ -319,6 +319,35 @@ def __call__(self):
     assert get_num_running_replicas(controller, A) == 2
 
 
+def test_smoothing_factor_with_0_replica():
+    """Unit test for smoothing_factor with 0 replica."""
+    config = AutoscalingConfig(
+        min_replicas=0,
+        max_replicas=2,
+        smoothing_factor=10,
+    )
+    policy = BasicAutoscalingPolicy(config)
+    new_num_replicas = policy.get_decision_num_replicas(
+        current_num_ongoing_requests=[],
+        curr_target_num_replicas=0,
+        current_handle_queued_queries=1,
+    )
+
+    # 1 * 10
+    assert new_num_replicas == 10
```

Comment:
If my understanding is correct, prior to your change this would have been `new_num_replicas == 1`?

Reply:
yes. (Still not perfect though).